src/test/java/atlastests/PredictionStepDefs.java

```diff
@@ -144,6 +149,11 @@ public void clickToExecutionSettingTabInPrediction() {
         $(By.xpath("//*[@class='pull-left']/ul/li[4]")).click();
     }
 
+    @When("^click to Execution Setting tab in Prediction without ee$")
+    public void clickToExecutionSettingTabInPredictionWithoutEE() {
+        $(By.xpath("//*[@id='filters']/div/div[2]/ul/li[4]/a")).click();
```

Comment:
It's better not to use constructions like this, in this way it would be stable:
  $$("#filters li").find(Condition.matchesText("Execution Settings")).click();